README.md

```diff
@@ -155,9 +155,10 @@ expect(AwesomeJob).to have_enqueued_job('Awesome', true)
 #### Testing scheduled jobs
 *Use chainable matchers `#at` and `#in`*
 ```ruby
-Awesomejob.perform_at 5.minutes.from_now, 'Awesome', true
+time = 5.minutes.from_now
```

Comment:
Could you please explain how this is a fix for time specification?

Reply:
It is really kind of trivial. Two calls of `5.minutes.from_now` may return different value because the current time is changing.

Reply:
Ah yes of course, that's great thanks.